http_stubs/tests/tests.py

```diff
@@ -48,8 +48,15 @@ def test_write_log(self, http_stub_factory, client):
         :param client: http client fixture
         """
         content_type = 'text/plain'
-        http_body = http_stub_factory(method=HTTPMethod.POST.name)
-        client.post('/default_path/', 'test', content_type=content_type)
+        http_body = http_stub_factory(
+            method=HTTPMethod.POST.name,
+            path='/regex/.*',
```

Comment:
Nit: `.*` part in the regexp is unnecessary: regexp `/regex/` will match same set of inputs as `/regex/.*` but will do it more efficiently as no time would be wasted for searching for the throwaway `.*` lookup.

Reply:
You're right) Thank you. Fixed